GX Map (#3)
genexuslabs/web-controls-library: src/components/map/readme.md

# gx-map

> Web Component used to display a map.

Reviewer: ```suggestion
An element for showing an interactive map based on [Leaflet](https://leafletjs.com/).
```

Author: Ready.

You can define the center, max zoom, and initial zoom of the map setting them as attribute of the component tag.
If you do not set any attribute, the map will initialize with default values.

<!-- Auto Generated Below -->

## Properties

| Property  | Attribute  | Description                            | Type     |
| --------- | ---------- | -------------------------------------- | -------- |
| `center`  | `center`   | The initial center of the map.         | `string` |

> | `maxZoom` | `max-zoom` | The max zoom scale aviable in the map. | `number` |

Reviewer: ```suggestion
| `maxZoom` | `max-zoom` | The max zoom level aviable in the map. | `number` |
```

Reviewer 2: available**

Author: Ready.

> | `zoom`    | `zoom`     | The initial zoom scale in the map.     | `number` |

Reviewer: ```suggestion
| `zoom`    | `zoom`     | The initial zoom level in the map.     | `number` |
```

Author: Ready.

# gx-map

> An element for showing an interactive map build with [LeafletJS](https://leafletjs.com/).

Reviewer: ```suggestion
An element for showing an interactive map built using [LeafletJS](https://leafletjs.com/).
```

Author: Ready!

> You can define the center, max zoom, and initial zoom of the map setting them as attribute of the component tag.

Reviewer: ```suggestion
You can define the center, max zoom, and initial zoom of the map setting them as attributes of the component tag.
```

Author: Ready!
